subprojects/docs/src/samples/task-with-arguments/projectInfoTask.sample.conf

```diff
@@ -0,0 +1,36 @@
+commands: [{
+    execution-subdirectory: groovy
+    executable: gradle
+    args: --include-build project-info-plugin projectInfo --format plain
+    expected-output-file: plain.out
+},
+{
+    execution-subdirectory: groovy
+    executable: gradle
+    args: --include-build project-info-plugin --rerun-tasks projectInfo --format json
```

Comment:
❓ You are using the `--rerun-tasks` flag because the test reuse the same environment? If this is the case, maybe we should just use `-q` instead to silence the lifecycle task progress.